packages/flutter/lib/src/material/slider.dart

```diff
@@ -673,8 +686,10 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     sliderTheme = sliderTheme.copyWith(
       trackHeight: sliderTheme.trackHeight ?? _defaultTrackHeight,
       activeTrackColor: widget.activeColor ?? sliderTheme.activeTrackColor ?? theme.colorScheme.primary,
+      secondaryValueTrackColor: widget.secondaryValueTrackColor ?? sliderTheme.secondaryValueTrackColor ?? theme.colorScheme.primary.withOpacity(0.62),
```

Comment:
@HansMuller Hi, I am not sure how to confirm the default value, whom I can ask help? Thanks :)